### PR TITLE
Allow specification of ACL when creating signed url for upload

### PR DESCRIPTION
**Pull request by publicmayhem**
I would like to use s3-beam to create signed urls with acl's that differ from public-read, such as authenticated-read. I've made the necessary small mod to allow this. Can you review and include? Thx

### CODE REVIEW

File: src/clj/s3_beam/handler.clj [hunk 1]
```diff
@@ -48,14 +49,14 @@
    "sa-east-1"      "s3-sa-east-1"})
 
 (defn sign-upload [{:keys [file-name mime-type]}
-                   {:keys [bucket aws-zone aws-access-key aws-secret-key]}]
+                   {:keys [bucket aws-zone aws-access-key aws-secret-key acl] :or {acl "public-read"}}]
```
Comment (martinklepsch) [inline src/clj/s3_beam/handler.clj]: There are two places where a `public-read` default is provided. Probably one of both is enough?

Comment (publicmayhem) [inline src/clj/s3_beam/handler.clj]: Agreed, see latest change

Comment (publicmayhem) [inline src/clj/s3_beam/handler.clj]: Could you review my change for pull-merge consideration? Thx

**Comment by martinklepsch** [comment]
Here we go! Thanks for your efforts @publicmayhem & sorry for the long wait :)

**Pull request closed without merging**